### PR TITLE
Remove projects page and show workflows instead

### PR DESCRIPTION
**Pull request by wKich**
Replace the Projects view page with Workflows overview

Screenshot 2023-05-11 at 18 05 15

### CODE REVIEW

**Comment by dagda1** [comment]
@wKich  Should the text of this button change?

proj

**Comment by wKich** [comment]
@dagda1 It's according by a design https://www.figma.com/file/iu0HAOSoLOESA2GHyxuHLg/parados-mui?type=design&node-id=3641-4279&t=WdZCjlvqc9QnAzNs-4

File: plugins/parodos/src/components/workflow/WorkflowsTable.tsx [hunk 1]
```diff
@@ -58,13 +93,46 @@ const formatDate = new Intl.DateTimeFormat('en', {
...
+      return;
+    }
+
+    setOpenFilter(false);
+  };
+  const handleChangeFilter = (
+    event: React.MouseEvent<Node, MouseEvent>,
+    filter?: ProjectWorkflow['workStatus'],
+  ) => {
+    handleFilterClose(event);
```
Comment (dagda1) [inline plugins/parodos/src/components/workflow/WorkflowsTable.tsx:121]: can we do this at once?

File: plugins/parodos/src/components/workflow/WorkflowsOverview.test.tsx [hunk 2]
```diff
@@ -53,7 +53,9 @@ describe('ProjectOverview', () => {
       // wait for re-render after receiving data
       await findByText('myProject');
       expect(
-        container.querySelector('td[value="In progress"]'),
+        container.querySelector(
+          'td[data-testid="511da8ce-4df7-438b-a9ec-0130f14884bd \'Running\'"]',
```
Comment (dagda1) [inline plugins/parodos/src/components/workflow/WorkflowsOverview.test.tsx:57]: can you not do getByText or some other way than this selector?

File: plugins/parodos/src/components/workflow/WorkflowsTable.tsx (hunk 1 above)
Comment (wKich) [reply plugins/parodos/src/components/workflow/WorkflowsTable.tsx:121]: What do you mean?

Comment (dagda1) [reply plugins/parodos/src/components/workflow/WorkflowsTable.tsx:121]: I thought it was 2 setState calls.   It's only 1.  My bad.

**Pull request merged**